src/alto-ui/Form/TagInput/TagInput.scss

```diff
@@ -0,0 +1,49 @@
+@import '../../scss/inc';
+
+.TagInput__tags-list {
+  display: flex;
```

Comment:
Component height "jumps" when user enter the tag

![input_tag](https://user-images.githubusercontent.com/5168044/54666356-d4d3da80-4ae9-11e9-98c6-7c082285d274.gif)

Reply:
yes I know. Not a big issue right, will fix it if needed later.